apollo-configservice/src/test/java/com/ctrip/framework/apollo/configservice/integration/AbstractBaseIntegrationTest.java

```diff
@@ -46,7 +48,8 @@
 
   private static final Gson GSON = new Gson();
 
-  protected RestTemplate restTemplate = (new TestRestTemplate()).getRestTemplate();
+  protected RestTemplate restTemplate = (new TestRestTemplate(new RestTemplateBuilder()
+      .setConnectTimeout(Duration.ofSeconds(5)))).getRestTemplate();
```

Comment:
Why we set the connect timeout here? Do we also need to set the read timeout?

Reply:
it seems that the cost of the initialization increased break the unit test by accident. there is no need to increased the read timeout, because the processing efficiency is not come down.